csrc/id_model/indexing.cpp

```diff
@@ -1144,7 +1152,7 @@ std::vector<PredicateInfo> TensorIndexer::getPredicates(
     info.loop_stage_ = loop_stage;
 
     info.start_predicate_ = SimplifyingIrBuilder::geExpr(
-        SimplifyingIrBuilder::addExpr(start_idx, info.start_offset_), zero_val);
+        SimplifyingIrBuilder::addExpr(start_idx, zero_val), zero_val);
```

Comment:
Why is this change?

Reply:
Not necessary. It was a debugging change.